arch/arm/arm64/include/uk/asm/lcpu.h

```diff
@@ -381,4 +381,9 @@ static inline void ukarch_spinwait(void)
 	/* Intelligent busy wait not supported on arm64. */
 }
 
+static inline void ukarch_dbg_break(void)
```

Comment:
Eventually this function shouldn't even be visible publicly. `ukdebug` should just register one function that runs at the end of the early init tab and one function that runs at the end of the (late) inittab. Both of these functions include a breakpoint and allow gdb to connect to the stub. We already do this in case of the inittab. We can't do it in case of the early init tab because the functions in the early init tab are executed before the lcpu and interrupts are initialized. That's why I manually inserted the early breakpoint in `ukboot` (at the earliest possible point in the boot sequence). If the lcpu and interrupts can be initialized early enough, we can make `ukarch_dbg_break` private to `ukdebug`.

If you want me to remove the definition in `arch/` now, I can create a new public header in `ukdebug` that contains the `ukarch_dbg_break` definition. Or we wait until `ukarch_dbg_break` becomes private and then remove the definition in `arch/`.